models/job.go

```diff
@@ -58,6 +58,7 @@ type JobSpec struct {
 	Dependencies map[string]JobSpecDependency // job name to dependency
 	Assets       JobAssets
 	Hooks        []JobSpecHook
+	Project      ProjectSpec
```

Comment:
Why is this added?

Reply:
have added this because when getting Replay from the repository, need to have Job and Project. second thought, i will remove this and use project from user input instead

Reply:
another usage: on the replay syncer, we need to know the project of each active replays that need to be synced, so in the replay repository, we fetch Replay that got preloaded with Job and Project (that is why Project is added). Thoughts?

Reply:
yes, i can remove this and call project where it is needed (when calling scheduler)